res/Sectors/M1105/hkakhaeaw.tab

```diff
@@ -10,7 +10,7 @@ Hkak	E	0712	Kaalakhi 0712	X445000-0		Ba		004	NaXX	F7 V	{ -3 }	(700-5)	[0000]		14
 Hkak	F	0913	Hkear 0913	X8A0000-0		Ba He		003	NaXX	K5 V M7 V	{ -3 }	(600-5)	[0000]		11	-30
 Hkak	F	0920	Hkear 0920	X470000-0		Ba De He		002	NaXX	G0 V M4 V	{ -3 }	(900-5)	[0000]		8	-45
 Hkak	F	1115	Hkear 1115	X73A000-?		Di Wa		024	NaXX	G5 V M7 V	{ -3 }	(500-5)	[0000]		11	-25
-Hkak	F	1215	Hkear 1215	X765000-?		Di(Dawn) Ga		005	NaXX	F5 V	{ -3 }	(C00-5)	[0000]		9	-60
+Hkak	F	1215	Hkear 1215	X765000-?		Di Ga (Dawn)		005	NaXX	F5 V	{ -3 }	(C00-5)	[0000]		9	-60
```

Comment:
Please revert this - `Di(xxxx)` is a single code. See https://travellermap.com/doc/secondsurvey.html#remarks